frontend/packages/helm-plugin/src/actions/creators.ts

```diff
@@ -5,7 +5,7 @@ import { deleteResourceModal } from '@console/shared';
 import { HelmActionsScope } from './types';
 
 export const getHelmDeleteAction = (
-  { releaseName, namespace, redirect }: HelmActionsScope,
+  { obj: { name: releaseName, namespace }, redirect }: HelmActionsScope,
```

Comment:
Can we use a better name than `obj`. This represents a helm release? Maybe just call it `release` even if it is a partial.